website/docs/components/form/file-input/partials/accessibility/accessibility.md

```diff
@@ -0,0 +1,15 @@
+## Conformance rating
```

Comment:
I think it's safe to pre-populate this with what we have for the Text Input then we can ask Melanie if there's anything else we'd need to add.